Dockerfile

```
@@ -1,13 +1,18 @@
 ARG PHP_VERSION=8.1.12-fpm-alpine
+ARG COMPOSER_TAG_VERSION=2-bin
```

```suggestion
ARG COMPOSER_TAG_VERSION=2.4.4-bin
```
Why not exact version?
ref: https://hub.docker.com/layers/composer/composer/2.4.4-bin/images/sha256-1fcc14e3597f38f171393b91f0a16e55aac38d1479e44368669da93c4d0ddc13?context=explore

---

Didn't notice that this tag is available.